package.json

```diff
@@ -722,6 +722,26 @@
           "markdownDescription": "`#editor.lineNumbers#` is determined by the active Vim mode, absolute when in insert mode, relative otherwise.",
           "default": false
         },
+        "vim.targets.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable [targets.vim](https://github.com/wellle/targets.vim#quote-text-objects) plugin (not fully implmeneted yet).",
+          "default": false
+        },
+        "vim.targets.smartQuotes.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable smart quotes movements",
+          "default": true
+        },
+        "vim.targets.smartQuotes.breakThroughLines": {
+          "type": "boolean",
+          "markdownDescription": "Whether to break through lines when using [n]ext/[l]ast motion.",
```

Comment:
What does this mean by `[n]ext/[l]ast motion`?

Reply:
It described in targets.vim.
we have new motions: din", which stands for "delete in next double-quotes" and dil" which stands for "delete in last double-quotes".

Reply:
Can you link to that in the description as well?

Reply:
```suggestion
          "markdownDescription": "Whether to break through lines when using [n]ext/[l]ast motion, see [targets.vim#next-and-last-quote](https://github.com/wellle/targets.vim#next-and-last-quote).",
```